stafeev_artem/task_02.cpp

```diff
@@ -0,0 +1,75 @@
+#include <iostream>
+#include <vector>
+
+enum { kVerticesCount = 13 };
+
+class Graph {
+  using VertexId = int;
+  using EdgeId = int;
+
+ public:
+  void add_vertex() {}
+
+  void add_edge(VertexId from_vertex_id, VertexId to_vertex_id) {
+    edges.emplace(edges.end(), (Edge(count++, from_vertex_id, to_vertex_id)));
```

Comment:
1) Лучше просто `emplace_back` использовать
```suggestion
    edges.emplace_back(count++, from_vertex_id, to_vertex_id);
```
2) Лучше выделить логику по генерации нового уникального id в отдельную функцию, например
```cpp
EdgeId get_new_edge_id() {
  return count++;
}
```